app/src/main/java/com/lateinit/rightweight/ui/routine/editor/RoutineExerciseAdapter.kt

```diff
@@ -75,7 +75,9 @@ class RoutineExerciseAdapter(
             binding.textViewExercisePart.setText(exercisePartName, false)
 
             binding.recyclerViewSet.adapter = routineSetAdapter
+            binding.recyclerViewSet.setItemAnimator(null)
```

Comment:
```suggestion
            binding.recyclerViewSet.itemAnimator = null
```
여기도 다른 곳이랑 같이 이렇게 해도 되지 않을까요?

Reply:
아 그렇군요! 방금 수정했습니다 감사합니다 👍